apps/test-can/include/CANConfig.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "CANInterface.h"
+#include "CANMsg.h"
+#include "hw_bridge.h"
+
+static mbed_error_status_t handle_test_msg_one_shot(CANMsg& msg);
+
+namespace CANConfig {
+
+using namespace HWBRIDGE;
+
+static CANMsgMap rxStreamedMsgMap = {
+    // Msg 1
+    {COMMON_TEST_MESSAGE2,
+     {
+         {COMMON_TEST_SIGNAL2, UWRT_MARS_ROVER_CAN_COMMON_TEST_MESSAGE2_COMMON_TEST_SIGNAL2_TEST_VALUE_2_CHOICE},
+         {COMMON_TEST_SIGNAL3, UWRT_MARS_ROVER_CAN_COMMON_TEST_MESSAGE2_COMMON_TEST_SIGNAL3_SNA_CHOICE},
+     }},
+
+    // Msg 2
+};
+
+static CANMsgMap txStreamedMsgMap = {
+    // Msg 1
+    {COMMON_TEST_MESSAGE2,
+     {
+         {COMMON_TEST_SIGNAL2, UWRT_MARS_ROVER_CAN_COMMON_TEST_MESSAGE2_COMMON_TEST_SIGNAL2_TEST_VALUE_2_CHOICE},
+         {COMMON_TEST_SIGNAL3, UWRT_MARS_ROVER_CAN_COMMON_TEST_MESSAGE2_COMMON_TEST_SIGNAL3_SNA_CHOICE},
+     }},
+
+    // Msg 2
+};
+
+const static CANMsg::CANMsgHandlerMap rxOneShotMsgHandler = {
+    {COMMON_TEST_MESSAGE1, &handle_test_msg_one_shot},
+};
```

Comment:
the CANMsgHandlerMap is the same as how we used it before, but this is only for received one-shot messages